dev-libs/glib/glib2-2.54.3.recipe

```diff
@@ -65,48 +64,62 @@ PROVIDES_devel="
 	cmd:gobject_query$secondaryArchSuffix
 	cmd:gtester$secondaryArchSuffix
 	cmd:gtester_report$secondaryArchSuffix
-	devel:libgio_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	devel:libglib_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	devel:libgmodule_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	devel:libgobject_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
-	devel:libgthread_2.0$secondaryArchSuffix = 0.5400.2 compat >= 0
+	devel:libgio_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
+	devel:libglib_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
+	devel:libgmodule_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
+	devel:libgobject_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
+	devel:libgthread_2.0$secondaryArchSuffix = 0.5400.3 compat >= 0
 	"
-
 REQUIRES_devel="
 	glib2$secondaryArchSuffix == $portVersion base
-	devel:libffi$secondaryArchSuffix
-	devel:libiconv$secondaryArchSuffix
-	devel:libintl$secondaryArchSuffix
-	devel:libpcre$secondaryArchSuffix
-	devel:libz$secondaryArchSuffix
-	cmd:perl
 	haiku$secondaryArchSuffix
-	lib:libffi$secondaryArchSuffix
+	cmd:perl
 	lib:libiconv$secondaryArchSuffix
 	lib:libintl$secondaryArchSuffix
+	lib:libffi$secondaryArchSuffix
 	lib:libpcre$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libffi$secondaryArchSuffix
+	devel:libpcre$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
 	"
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:libiconv$secondaryArchSuffix
-	devel:libz$secondaryArchSuffix
 	devel:libffi$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
 	devel:libintl$secondaryArchSuffix
 	devel:libpcre$secondaryArchSuffix >= 1
+	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
 	cmd:gcc$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
 	cmd:libtoolize$secondaryArchSuffix
-	cmd:aclocal
-	cmd:autoconf
 	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
 	cmd:python
 	cmd:which
 	"
 
+defineDebugInfoPackage glib2$secondaryArchSuffix \
+	$binDir/gapplication \
+	$binDir/gdbus \
+	$binDir/gio \
+	$binDir/gio-querymodules \
+	$binDir/glib-compile-resources \
+	$binDir/glib-compile-schemas \
+	$binDir/gresource \
+	$binDir/gsettings \
+	$libDir/libgio-2.0.so.0 \
+	$libDir/libglib-2.0.so.0 \
+	$libDir/libgmodule-2.0.so.0 \
+	$libDir/libgobject-2.0.so.0 \
+	$libDir/libgthread-2.0.so.0
```

Comment:
If I'm not mistaken, it would be better to pass the names of the real files, not the symlinks.
```diff
-	$libDir/libgio-2.0.so.0 \
+	$libDir/libgio-2.0.so.$libVersion \
-	$libDir/libglib-2.0.so.0 \
+	$libDir/libglib-2.0.so.$libVersion \
-	$libDir/libgmodule-2.0.so.0 \
+	$libDir/libgmodule-2.0.so.$libVersion \
-	$libDir/libgobject-2.0.so.0 \
+	$libDir/libgobject-2.0.so.$libVersion \
-	$libDir/libgthread-2.0.so.0
+	$libDir/libgthread-2.0.so.$libVersion
```